### PR TITLE
Pba 4760

### PR DESCRIPTION
**Pull request by pcaseooyala**
Added platform support information to skin-schema.json for scrubberBar scrubberHandleColor, scrubberBar scrubberHandleBorderColor, adScrubberBar scrubberHandleColor and adScrubberBar scrubberHandleBorderColor.

### CODE REVIEW

File: skin-schema.json [hunk 1]
```diff
@@ -597,11 +597,19 @@
             },
             "scrubberHandleColor": {
               "description": "The color of the draggable scrubber object in the scrubber bar. Any valid CSS color value (named colors, three-digit hex color, six-digit hex color, RGB colors).",
-              "type": "string"
+              "type": "string",
+			  "platform support": {
```
Comment (lental) [inline skin-schema.json]: can you switch to spaces instead of tabs for this file?

Comment (pcaseooyala) [inline skin-schema.json]: Sure thing

Comment (pcaseooyala) [inline skin-schema.json]: Pushed tab->spaces change

**Comment by lental** [comment]
@pilievOoyala @aldigun @mistermendez FYI.  We'll roll out the rest of the json later, but this is enough for the new elements

LGTM

**Pull request merged**